xds/internal/xdsclient/xdsresource/errors.go

```diff
@@ -37,6 +37,10 @@ const (
 	// ErrorTypeResourceTypeUnsupported indicates the receipt of a message from
 	// the management server with resources of an unsupported resource type.
 	ErrorTypeResourceTypeUnsupported
+	// ErrorTypeIgnored indicates the error received from the management server
+	// after at least one successful read.These errors are not propagated to
+	// watchers in the xDS data model layer.
+	ErrorTypeIgnored
```

Comment:
The name `ErrorTypeIgnored`, does not say anything about why the error is being ignored. Actually the fact that the error is ignored should not even matter here. We might change the implementation tomorrow to not ignore the error. So, a name which describes the actual error would be better.

How about `ErrTypeStreamFailedAfterRecv` and the comment as follows:
```
// ErrTypeStreamFailedAfterRecv indicates an ADS stream error, after
// successful receipt of at least one message from the server. 
```

Reply:
I didnt think of it that way. Im updating this per comment